ast/builtins.go

```diff
@@ -2458,8 +2458,8 @@ var GraphQLParse = &Builtin{
 	Description: "Returns AST objects for a given GraphQL query and schema after validating the query against the schema. Returns undefined if errors were encountered during parsing or validation.",
```

Comment:
Perhaps we could add parameter descriptions? Or would you think it's obvious that it's either parsed from the string to used as-is from the object? 🤔

Reply:
I think embellishing the descriptions here is worthwhile. I'll add some improved descriptions in shortly!

Reply:
I have added notes to each of the 3x affected builtins that query/schema can be either strings or GraphQL AST objects. :+1: